include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -1482,10 +1482,21 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
 
             void write(ThreadParams* threadParams, std::string mpiTransportParams)
             {
-                /* y direction can be negative for first gpu */
-                const pmacc::Selection<simDim> localDomain = Environment<simDim>::get().SubGrid().getLocalDomain();
-                DataSpace<simDim> particleOffset(localDomain.offset);
-                particleOffset -= threadParams->window.globalDimensions.offset;
+                const SubGrid<simDim>& subGrid = Environment<simDim>::get().SubGrid();
+                const pmacc::Selection<simDim> localDomain = subGrid.getLocalDomain();
+                /* Offset to transform local particle offsets into total offsets for all particles within the
+                 * current window.
+                 * @attention A window can be the full simulation domain or the moving window.
+                 */
+                DataSpace<simDim> windowCellOffsetToTotalDomain(localDomain.offset);
```

Comment:
Maybe a bug: Most likely this must be `DataSpace<simDim> windowCellOffsetToTotalDomain(localDomain.offset+subGrid.getGlobalDomain().offset);`

Reply:
@psychocoderHPC I will test this.

Reply:
Checked - does not fix the problem (sim still running default LaserWakefield example `8.cfg`) 
For details on the plot, please see #4319
![grafik](https://user-images.githubusercontent.com/5121158/195647248-ac6a3d47-3053-4a0d-b050-e2f076eb043a.png)

Reply:
Full result looks as 
![grafik](https://user-images.githubusercontent.com/5121158/195869105-58b9860e-c127-4670-9b2e-c0dd867fcb46.png)

Thus is got actually worse than before.

Reply:
@psychocoderHPC I was looking at the definition here: https://github.com/ComputationalRadiationPhysics/picongpu/wiki/PIConGPU-domain-definitions . Is it still valid? 

If yes, what is with the `globalWindow.offset`?

Reply:
@PrometheusPi Yes the plot is still valid. I do not understand what you mean with `globalWindow.offset`?

Reply:
@psychocoderHPC What I meant was that I thought we compute the offset of the window (shaded area) and thus need `globalWindow.offset` (underlined in red below). 
But now I think what you calculate is the global position of one of the blue cells by adding the values `globalDomain.offset` and `localDomain.offset` (circled in red). 
![grafik](https://user-images.githubusercontent.com/5121158/200311822-0904c67d-1fd8-495e-aaf9-11cdc68bf883.png)

Is the "Window" not the output window we use for openPMD output?